src/store/DrawToolsStore.js

```diff
@@ -80,6 +86,36 @@ export default class DrawToolsStore {
         }
     };
 
+    onMouseMove = (e) => {
+        e.preventDefault();
+        const vectorType = this.stx.currentVectorParameters.vectorType;
+        const drawingMousePointer = document.getElementById('drawing-mouse-pointer');
+        const container = document.getElementsByClassName('chartContainer primary')[0];
+        const crosshairX = document.getElementsByClassName('stx_crosshair_x')[0];
+        const crosshairY = document.getElementsByClassName('stx_crosshair_y')[0];
```

Comment:
pls don't use `document` queries. SmartCharts is designed to be used as a library, so its scope needs to be confined within its own DOM.